data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -2935,8 +2935,7 @@ spec:
               configType:
                 description: Config type is the proxy configuration type. The valid values
                   are transparent or explicit
-                format: int32
-                type: integer
+                type: string
```

Comment:
This is a simple change to a file that you already changed. Please squash this commit.